packages/web-client/app/components/workflow-thread/postable/index.hbs

```diff
@@ -1,49 +1,59 @@
-{{#if (is-postable-on-new-day @postable)}}
-  <Boxel::DateDivider
-    @date={{@postable.timestamp}}
-    class="workflow-date-divider-animated"
-    data-test-date-divider
+{{#if @postable._isCheck}}
```

Comment:
It's usually a bad sign for how you're using inheritance/polymorphism if you need to have an `isKlass` kind of call like this.